test/rule-matches/label-content-name-mismatch-matches.js

```diff
@@ -0,0 +1,87 @@
+describe('label-content-name-mismatch-matches', function() {
```

Comment:
I don't think the following test case is addressed:
https://auto-wcag.github.io/auto-wcag/_testcases-embeds/SC2-5-3-label-content-name-mismatch_inapplicable_example_5.html